.travis.yml

```diff
@@ -2,7 +2,7 @@ sudo: required
 
 language: scala
 scala:
-  - 2.12.11
+#  - 2.12.11
```

Comment:
I think it's important to keep scala 2.12 compatibility for this public project for some more time

Reply:
Reenabled it